src/main/java/uk/gov/hmcts/reform/pip/data/management/models/court/CourtCsv.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.pip.data.management.models.court;
+
+import com.opencsv.bean.CsvBindAndSplitByName;
+import com.opencsv.bean.CsvBindByName;
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class captures the inbound Court CSV that will be ingested.
+ */
+@Data
+public class CourtCsv {
+
+    @CsvBindByName(column = "P&I ID")
+    private Integer uniqueId;
+
+    @CsvBindByName(column = "Court Desc")
+    private String courtName;
```

Comment:
Can you please change **courtName** to **name**? Frontend is using **name**.

Reply:
Updated to name :)